cloudsim-plus-testbeds/src/main/java/org/cloudsimplus/migration/DynamicVmCreationExample.java

```diff
@@ -37,19 +37,18 @@
 import org.cloudsimplus.builders.tables.CloudletsTableBuilderHelper;
 
```

Comment:
This example is not in fact creating VMs dynamically. Remove the file and add the other one you were working on.